src/main/java/net/tirasa/connid/bundles/googleapps/GoogleAppsConfiguration.java

```diff
@@ -192,6 +209,61 @@ public void setRemoveLicenseOnDisable(final boolean removeLicenseOnDisable) {
         this.removeLicenseOnDisable = removeLicenseOnDisable;
     }
 
+    @ConfigurationProperty(displayMessageKey = "proxyServerType.display",
```

Comment:
We should be using standard Java properties for HTTP proxy, not custom.

See https://docs.oracle.com/javase/6/docs/technotes/guides/net/proxies.html